snuba/reader.py

```diff
@@ -25,7 +25,12 @@
     Row = MutableMapping[str, Any]
     Result = TypedDict(
         "Result",
-        {"meta": Sequence[Column], "data": Sequence[Row], "totals": Row},
+        {
+            "meta": Sequence[Column],
+            "data": Sequence[Row],
+            "totals": Row,
+            "applied_sampling_rate": Optional[float],
```

Comment:
I also still don't think this attribute belongs here.

Reply:
Forgot to remove it. I moved it to RawQueryResult yesterday but I forgot to remove it from the TypedDict